frontend/src/App.js

```diff
@@ -57,26 +64,33 @@ function App() {
 								isLoaded &&
 								<>
 									<Route exact path="/">
-										<CenterFlexBox>
-											<HeaderTypography component="h1" variant="h5">
-												Coming soon!
-											</HeaderTypography>
-											<img src={logo} className="App-logo" alt="logo" />
-											<HeaderTypography component="h1" variant="h5">
-												NFT the world!
-											</HeaderTypography>
-											<Button
-												sx={{ mt: 2 }}
-												style={{ width: "min(50vw,500px)", minHeight: "min(30vh,150px)", fontSize: "4vh", backgroundColor: "#005438", borderRadius: "15px" }}
-												component={Link}
-												to="/drop"
-											>
-												JOIN THE DROP!
-											</Button>
-										</CenterFlexBox>
+										<Grid container>
+											<Grid item xs={8}>
+												<CenterFlexBox>
+													<HeaderTypography component="h1" variant="h5">
+														Coming soon!
+													</HeaderTypography>
+													<img src={logo} className="App-logo" alt="logo" />
+													<HeaderTypography component="h1" variant="h5">
+														NFT the world!
+													</HeaderTypography>
+													<Button
+														sx={{ mt: 2 }}
+														style={{ width: "min(50vw,500px)", minHeight: "min(30vh,150px)", fontSize: "4vh", backgroundColor: "#005438", borderRadius: "15px" }}
+														component={Link}
+														to="/drop"
+													>
+														JOIN THE DROP!
+													</Button>
+												</CenterFlexBox>
+											</Grid>
+											<Grid item xs={4}>
+												<AnnouncementPage user={user} isSidebar={true}/>
+											</Grid>
+										</Grid>
 									</Route>
 									<Route path="/faq">
-										<FaqPage/>
+										<FaqPage />
```

Comment:
Please avoid unnecessary git-diffs. Is this a problem with the ESlinter? If not, this should be undone.
```suggestion
										<FaqPage/>
```

Reply:
`npx eslint .` returned nothing for me

Reply:
Make sure you have `cd`ed into the frontend directoy before running the command.